### ISSUE TITLE
Issue when running testaudiocapture.c in ubuntu

### ISSUE THREAD

**Issue by yylittlefish** [emscripten-ports/SDL2#91]
Hello,
   I wanna use the sample code testaudiocapture.c to test audio record and play in ubuntu.
   I use "emcc testaudiocapture.c -s USE_SDL=2  -o index.html" to compile,
   I tried to run with firefox in ubuntu, the output and capture device could be detected and could call SDL_PauseAudioDevice functions, but I can't hear any sound when play :(
I also tried chrome in ubuntu, it shows "ERROR: Couldn't open an audio device for capture: No capture support!"
   I tested in windows using the same devices:
   when I run with edge in windows, the record and play works fine.
   but when I run with chrome in windows, it also can't find capture device.

 could you help to check in your side, please?

**Comment by Daft-Freak** [comment]
Hmm, works for me in Firefox and in Chrome after running `Module.SDL2.audioContext.resume()` in the console (autoplay policy workaround). (Both on Ubuntu)

**Comment by yylittlefish** [comment]
@Daft-Freak 
this is the console log, I should input the Module.SDL2.xx here, then click mouse button to record, right? but still can't hear sound..
INFO: Using audio driver: emscripten
INFO:  Capture device #0: 'System audio capture device'
INFO: Opening default playback device...
INFO: Opening capture device [[default]]...
INFO: Ready! Hold down mouse or finger to record!
Module.SDL2.audioContext.resume()   _<=input here, right?_
[object Promise]: {}

when I test "testaudiohotplug.c", I can hear sound.

**Comment by yylittlefish** [comment]
could you tell me the reason of running "Module.SDL2.audioContext.resume()"? is there any other way to do?

**Comment by yylittlefish** [comment]
@Daft-Freak Hi, after adding the resume logic, chrome could play sound normally(I tested the testaudiohotplug.c). but still can't record cause there's  error"No capture support!". 
do you know why?

**Comment by yylittlefish** [comment]
my chrome version is 76.0.3809.100

**Comment by Daft-Freak** [comment]
Looks like that error would be generated if `getUserMedia` is not supported. ([support check here](https://github.com/emscripten-ports/SDL2/blob/master/src/audio/emscripten/SDL_emscriptenaudio.c#L368)).

**Comment by yylittlefish** [comment]
@Daft-Freak yes, that's the reason! cause I access index.html through http , and chrome does not support "navigator.mediaDevices" in that case(should be changed to https or localhost access).

I have another question, I found SDL always detect and use default output/capture audio device 
event though I did not plugin any device. when I try to print device name, it is also not the real device name. so does it not support to choose output/capture device?
I share the log of running testaudioinfo.c:
INFO: Built-in audio drivers:
INFO:   0: emscripten
INFO:   1: disk
INFO:   2: dummy

INFO: Using audio driver: emscripten 

INFO: Found 1 output device:
INFO:   0: System audio output device

INFO: Found 1 capture device:
INFO:   0: System audio capture device


I
